README.md

```diff
@@ -253,8 +253,9 @@ component's "real" props. In this example lets use `_testID`.
 ```js
 let Greeting = props => <div>hello <strong _testID='name'>{props.name}</strong></div>;
 
-$(Greeting).render()
-  .prop({ name: 'Betty' })
+$(<Greeting />)
```

Comment:
Too true. :)  I'll make that change.

I do think there exists some confusion about the .prop vs .props still though.  Might be worth reviewing the other sections too.